doc/nrf/doc_styleguide.rst

```diff
@@ -57,6 +57,16 @@ It is also possible to define more than one default link text for a link, which
    .. _`Link text one`:
    .. _`Link text two`: http://..
 
+Breathe Domains
+===============
+
+In order to link to Doxygen content from RST, the following domains are to
+be used:
+
+* Function: ``:cpp:func:``
+* Structure: ``:c:type:``
+* Enum: ``:cpp:enumerator:``
```

Comment:
This one doesn't seem to work where it is currently used.

Reply:
yeah, the linking is broken for some reason, same happens upstream

Reply:
https://github.com/michaeljones/breathe/issues/437

Reply:
Can we add a comment that this does not currently work?